### PR TITLE
Karafecho patch 2

### PR DESCRIPTION
**Pull request by karafecho**
Edited search terms; commented out EPR/PEGS variables; corrected one minor typo.

### CODE REVIEW

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 1]
```diff
@@ -45,27 +45,27 @@ patient:
...
-    type: integer 
-  ROADTYPE:
-    categories:
-    - biolink:EnvironmentalExposure
-    enum:
-    - Rural Restricted Access
-    - Rural Unrestricted Access
-    - Urban Restricted Access
-    - Urban Unrestricted Access
-    type: string
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:68]: These three variables exist in covid patient data, so commenting them out in all features yaml file for covid will break the icees-db code when creating a new covid sqlite database. I think we will have to keep them to make the all features yaml file compatible with the current covid data.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 2]
```diff
@@ -783,20 +783,20 @@ patient:
...
-    - biolink:PhenotypicFeature
-    enum:
-    - Normal (18.5 to <25)
-    - Over (25 to <30)
-    - Obese (30+)
-    - Under (<18.5)
-    type: string
-    name_lookup:
-    - limit: 1
-      search_term: body mass index
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:799]: Same problem here - BMI_CATEGORY exists in patient data, so cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 3]
```diff
@@ -1028,33 +1028,33 @@ patient:
...
-    enum:
-    - <5
-    - 5-17
-    - 18-44
-    - 45-64
-    - 65-89
-    type: string
-    name_lookup:
-    - limit: 1
-      search_term: chronological age
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:1057]: Same comment as above - these variables cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 4]
```diff
@@ -1260,103 +1260,103 @@ patient:
...
-  DISTANCE2:
-    categories:
-    - biolink:ComplexChemicalMixture
-    - biolink:EnvironmentalExposure
-    maximum: 6
-    minimum: 1
-    type: integer
-    name_lookup:
-    - limit: 1
-      search_term: roadway exposure
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:1359]: Same here - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 5]
```diff
@@ -1472,132 +1472,132 @@ patient:
...
-    categories:
-    - biolink:PhenotypicFeature
-    enum:
-    - Non-Hispanic/Non-Latino
-    - Unknown/NR
-    - Hispanic Latino
-    name_lookup:
-    - limit: 1
-      search_term: ethnic group
-    type: string
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:1600]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 6]
```diff
@@ -2045,16 +2045,16 @@ patient:
...
-  GENDER:
-    categories:
-    - biolink:PhenotypicFeature
-    enum:
-    - Female
-    - Male
-    type: string
-    name_lookup:
-    - limit: 1
-      search_term: biological sex
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:2057]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 7]
```diff
@@ -2202,20 +2202,20 @@ patient:
...
-    - '0'
-    - '1'
-    type: string
-  IN_FINAL_SAMPLE:
-    categories:
-    - biolink:PhenotypicFeature
-    enum:
-    - 'True'
-    - 'False'
-    type: string
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:2218]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 8]
```diff
@@ -2414,24 +2414,24 @@ patient:
...
-      search_term: household income
-  MEDIANHOUSEHOLDINCOME_qcut:
-    categories:
-    - biolink:EnvironmentalExposure
-    maximum: 4
-    minimum: 1
-    type: integer
-    name_lookup:
-    - limit: 1
-      search_term: household income
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:2434]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 9]
```diff
@@ -2740,38 +2740,38 @@ patient:
...
-    - biolink:ChemicalEntity
-    - biolink:EnvironmentalExposure
-    type: integer
-  ORIGINAL_ANALYTIC_SAMPLE:
-    categories:
-    - biolink:PhenotypicFeature
-    enum:
-    - 'True'
-    - 'False'
-    type: string
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:2774]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 10]
```diff
@@ -2818,31 +2818,31 @@ patient:
...
-    minimum: 1
-    name_lookup:
-    - limit: 1
-      search_term: particulate matter
-    type: integer
-  PM25_N_OBS:
-    categories:
-    - biolink:ChemicalEntity
-    - biolink:EnvironmentalExposure
-    type: integer
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:2845]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 11]
```diff
@@ -3035,57 +3035,57 @@ patient:
...
-      search_term: racial group
-  RESPONDER_STATUS:
-    categories:
-    - biolink:PhenotypicFeature
-    enum:
-    - Neither
-    - Hyper
-    - Hypo
-    - Super Hypo
-    type: string
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:3088]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 12]
```diff
@@ -3188,60 +3188,60 @@ patient:
...
-    - D
-    type: string
-  SNP4:
-    categories:
-    - biolink:SequenceVariant
-    enum:
-    - N
-    - L
-    - M
-    type: string
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:3244]: Same as above - they cannot be commented out here.

File: config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml [hunk 13]
```diff
@@ -3502,93 +3502,93 @@ patient:
...
-  TLR4_DIST_3X_qcut:
-    categories:
-    - biolink:ComplexChemicalMixture
-    - biolink:EnvironmentalExposure
-    maximum: 5
-    minimum: 1
-    name_lookup:
-    - limit: 1
-      search_term: roadway exposure
-    type: integer
```
Comment (hyi) [inline config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml:3591]: Same as above - they cannot be commented out here.

**Review by hyi — COMMENTED** [review]
@karafecho I left some comments on covid all features yaml file. Basically all those variables cannot be commented out since they do exist in covid patient data.

**Comment by karafecho** [comment]
But your slack message indicated that we don't include the final EPR/PEGS join step for the covid and pcd datasets?

**Comment by hyi** [comment]
Yeah, that would work. I can just write a script to remove those commented-out variables from covid data. Then this PR looks OK to me, but I'll defer the final approval of this PR to Max.

**Review by hyi — APPROVED** [review]

**Comment by karafecho** [comment]
The next time we rerun FHIR PIT, which should be relatively soon, given the code extensions to support new/updated exposures data, then I think we can just skip the final EPR/PEGS join.

**Comment by hyi** [comment]
That's right, I don't have to remove these variables from current COVID data given that we are going to rerun FHIR PIT to generate new COVID data soon.

**Review by maximusunc — COMMENTED** [review]
In general, I think the way you commented out the search term is ok and my generation script will be fine, but I would have just commented out the whole name_lookup block.

**Review by maximusunc — CHANGES_REQUESTED** [review]
Sorry if I was unclear earlier. I think the whole `name_lookup` block should be commented out.

# name_lookup:
#  -limit: 1
#   search_term: salbutamol

**Comment by karafecho** [comment]
No, I understood your suggestion; I just thought that it might be easier to comment out the `search_term` only, so that if I want to include a commented-out `search_term` for some future reason, then I'd only have to remove the # from the `search_term`, not the whole block.

That said, if it makes things cleaner to comment out the whole block, then I can do that.

FWIW, as much as I hate these YAML files, I have been feeling good about cleaning them up and aligning them with project goals, so if commenting out the whole block is best practice, then we should do that.

**Comment by maximusunc** [comment]
Yeah, I think it's cleaner and clearer if the whole block is commented out. If you want, I can make that change and use some keyboard shortcuts to do it all at once.

**Comment by karafecho** [comment]
Yes, please. If you know how to apply keyboard shortcuts to comment out the whole block all it once, then that would be great. I was using find-and-replace, but that isn't a very efficient approach.

**Review by maximusunc — APPROVED** [review]

**Review by hyi — APPROVED** [review]

**Pull request merged**